conda/recipes/cuml/meta.yaml

```diff
@@ -31,11 +31,13 @@ requirements:
     - cmake>=3.14
     - cudf {{ minor_version }}
     - libcuml={{ version }}
+    - libcumlprims=0.10
```

Comment:
Can this (and the other below it) use `{{ version }}` instead?

Reply:
Yeah, there weren’t because at the beginning of 0.10 some package (I think sask CUDA) were lagging behind